styles/wiki.nixos.org/catppuccin.user.css

```diff
@@ -0,0 +1,1480 @@
+      color: @text !important;
+    }
+
+    td[style*="background-color"],
+    td[style*="background-color"] * {
+      color: @crust !important;
+    }
+
+    #toc-Services > a:nth-child(1) > div:nth-child(1) {
+      color: @text !important;
+    }
+
+    .cdx-button.cdx-button--fake-button--enabled.cdx-button--weight-quiet.cdx-button--action-progressive,
+    .vector-menu-tabs .mw-list-item a,
+    .vector-feature-zebra-design-enabled .vector-toc .vector-toc-link,
+    .mw-parser-output a.extiw,
+    .mw-parser-output a.external,
+    .mw-collapsible-toggle-default .mw-collapsible-text,
+    .vector-feature-zebra-design-enabled
+      .vector-pinnable-element
+      .mw-list-item
+      a,
+    .vector-feature-zebra-design-enabled
+      .vector-dropdown-content
+      .mw-list-item
+      a,
+    .vector-feature-zebra-design-enabled
+      .vector-pinnable-element
+      .mw-list-item
+      a:not(.mw-selflink):visited,
+    .vector-feature-zebra-design-enabled
+      .vector-dropdown-content
+      .mw-list-item
+      a:not(.mw-selflink):visited,
+    .uls-language-block a,
+    .lang-list-button,
+    .fancycaptcha-reload,
+    #pt-userpage-2 a:not(.mw-selflink),
+    .vector-pinnable-element .mw-list-item a,
+    .vector-pinnable-element .mw-list-item a:not(.mw-selflink):visited,
+    .vector-toc .vector-toc-link,
+    .oo-ui-buttonElement-frameless.oo-ui-widget-enabled.oo-ui-flaggedElement-progressive
+      > .oo-ui-buttonElement-button,
+    .oo-ui-buttonElement-frameless.oo-ui-widget-enabled.oo-ui-flaggedElement-progressive
+      > .oo-ui-buttonElement-button:hover,
+    .vector-dropdown-content .mw-list-item a,
+    .vector-dropdown-content .mw-list-item a:not(.mw-selflink):visited {
+      color: @accent-color;
+    }
+
+    .cdx-button:enabled.cdx-button--weight-primary.cdx-button--action-progressive,
+    .cdx-button.cdx-button--fake-button--enabled.cdx-button--weight-primary.cdx-button--action-progressive,
+    .mw-ui-button.mw-ui-progressive:not(:disabled),
+    .mw-ui-button.mw-ui-progressive:not(:disabled):hover {
+      background-color: @accent-color;
+      border-color: @accent-color;
+      color: @base;
+    }
+
+    .mw-message-box-warning,
+    .boilerplate {
+      border-color: @accent-color !important;
+      background-color: fade(@accent-color, 25%) !important;
+      color: @text;
+    }
+
+    .vector-feature-zebra-design-enabled .vector-sticky-pinned-container::after,
+    .vector-sticky-pinned-container::after {
+      background: none;
+    }
+
+    .vector-feature-zebra-design-enabled .vector-pinnable-header-toggle-button {
+      background-color: @surface0;
+      color: @text;
+      &:hover {
+        background-color: @base;
+      }
+    }
+
+    .vector-feature-zebra-design-enabled .vector-pinnable-header,
+    .vector-pinnable-header,
+    .vector-pinnable-element .vector-menu-heading {
+      border-bottom-color: @surface0;
+    }
+
+    hr {
+      border-bottom-color: @surface2;
```

Comment:
I believe this should be background color (or at least thats how the one on https://wiki.nixos.org/w/index.php?title=NixOS_Wiki&action=edit is themed).

Reply:
Still is border-bottom-color but now with a different color, I believe it should be `background-color`.